src/it/java/net/spy/memcached/ClientBaseCase.java

```diff
@@ -88,6 +88,9 @@ public static void setUpConfigEndpoint() throws Exception {
   
   @Before
   public void setUp() throws Exception {
+    if (TestConfig.getInstance().getClientMode() == ClientMode.Dynamic && TestConfig.isTlsMode()) {
+      restoreClusterConfigForTLS(TestConfig.PORT_NUMBER);
+    }
```

Comment:
Why do you need to do this? Dynamic mode tests didn't work before with non-TLS?

Reply:
It worked before with non-TLS. The restoreClusterConfigForTLS() method is to configure the local server for Dynamic mode (pretend the local server is an Elasticache cluster for Auto discovery feature testing).